default.nix

```diff
@@ -10,7 +10,25 @@
 , iosSdkVersion ? "10.2"
 }:
 let iosSupport = system != "x86_64-darwin";
-    globalOverlay = self: super: { };
+    globalOverlay = self: super: {
+
+      # need to override cabal2nix to avoid evaluation errors on Android.
+      # https://github.com/NixOS/cabal2nix/pull/344
+      haskellPackages = super.haskellPackages.override {
+        overrides = self_: super_: {
+          cabal2nix = super_.cabal2nix.overrideAttrs (drv: {
```

Comment:
This should probably be `overrideCabal`, right?

Reply:
Updated